DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/cleanup_utils.py

```diff
@@ -114,3 +115,53 @@ async def cleanup_sysctl():
     logger = AppLogger(DEFAULT_LOGGER)
     logger.info('Restoring sysctl values')
     await RecoverableSysctl.recover()
+
+
+async def cleanup_kbyte_per_sec_rate_value(dev, all_values=False, bc=False, unk_uc=False, unreg_mc=False):
+    """
+    Restore values changed during test viz:
+        - all kbyte_per_sec_rate values
+        - bc_kbyte_per_sec_rate values
+        - unk_uc_kbyte_per_sec_rate
+        - unreg_mc_kbyte_per_sec_rate
+    Can be used separately or by using `cleanup_kbyte_per_sec_rate_value` fixture.
+    """
+    logger = AppLogger(DEFAULT_LOGGER)
+    logger.info('Restoring kbyte_per_sec_rate values')
+    out = await DevlinkPort.show(
+        input_data=[{dev.host_name: [{'options': '-j'}]}],
+        parse_output=True)
+    devlink_entries = out[0][dev.host_name]['parsed_output']
+
+    # restoring kbyte_per_sec_rate all values
+    if all_values:
+        input_data = ({dev.host_name: [
+            {'dev': f'{device}', 'name': f"{item['name']}", 'value': '0', 'cmode': 'runtime'}]}
+            for device in devlink_entries['param']
+            for item in devlink_entries['param'][device]
+        )
+        await DevlinkPort.set(input_data=input_data)
```

Comment:
Add return here so that you don't go into other conditions after you reset all values